File: aspnet-in-practice/05/VB/Display/App_Code/ApplicationObjectContext.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports ObjectModel

Public Class ApplicationObjectContext
	Public Shared ReadOnly Property Current() As NorthwindEntities
		Get
			Return TryCast(HttpContext.Current.Items("ObjectContext"), NorthwindEntities)
		End Get
	End Property
End Class

File: aspnet-in-practice/05/CS/Display/App_Code/ApplicationObjectContext.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using ObjectModel;

public class ApplicationObjectContext
{
	public static NorthwindEntities Current
	{
		get
		{
			return HttpContext.Current.Items["ObjectContext"] as NorthwindEntities;
		}
	}
}